codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/Conventions.kt

```diff
@@ -53,11 +54,29 @@ public class MessageOrEnumConvention(ts: TypeSystem) : BaseJavaTypeConvention(ts
  *
  * @throws IllegalStateException if the type name is unknown.
  */
-public class MessageOrBuilderConvention(ts: TypeSystem) : BaseJavaTypeConvention(ts) {
+public class MessageOrBuilderConvention(ts: TypeSystem) : BaseJavaConvention<TypeName>(ts) {
 
     override fun declarationFor(name: TypeName): Declaration<Java, ClassName> {
         val decl = MessageOrEnumConvention(typeSystem).declarationFor(name)
         val messageOrBuilder = decl.name.withSuffix("OrBuilder")
         return Declaration(messageOrBuilder, messageOrBuilder.javaFile)
     }
 }
+
+/**
+ * This convention governs declarations of generated gRPC stubs.
```

Comment:
How about making it more uniform?

"The convention governing declarations of generated gRPC stubs."